cli/app.go

```diff
@@ -170,8 +174,9 @@ var app = &cli.App{
 					Action: ListOrganizationsAction,
 				},
 				{
-					Name:  "api-key",
-					Usage: "work with an organization's api keys",
+					Name:      "api-key",
+					Usage:     "work with an organization's api keys",
+					UsageText: createUsageText("organizations api-key", []string{apiKeyCreateFlagOrgID}, true),
```

Comment:
i could see cases where showcasing optional flags might be helpful - in a data query API for example. however, what you have is totally reasonable

Reply:
Yeah it would be great if there were really in depth `Description`s like the one for [`module upload`](https://github.com/viamrobotics/rdk/blob/main/cli/app.go#L1078-L1089) for everything, but I may leave it to respective teams (data) to determine if they need to showcase a few, key optional flags. In any case, running something like `viam data export --help` or misusing `viam data export` will print a longer help message with more than just the `UsageText` that does include all options:
```
NAME:
   viam data export - download data from Viam cloud

USAGE:
   viam data export --destination=<destination> --data-type=<data-type> [other options]

OPTIONS:
   --bbox-labels value                       bbox labels filter. accepts string labels corresponding to bounding boxes within images  (accepts multiple inputs)
   --component-name value                    component name filter
   --component-type value                    component type filter
   --data-type value                         data type to be downloaded: either binary or tabular
   --destination value                       output directory for downloaded data
   --end value                               ISO-8601 timestamp indicating the end of the interval filter
   --location-ids value                      locations filter  (accepts multiple inputs)
   --machine-id value, --robot-id value      machine id filter
   --machine-name value, --robot-name value  machine name filter
   --method value                            method filter
   --mime-types value                        mime types filter  (accepts multiple inputs)
   --org-ids value                           orgs filter        (accepts multiple inputs)
   --parallel value                          number of download requests to make in parallel (default: 100)
   --part-id value                           part id filter
   --part-name value                         part name filter
   --start value                             ISO-8601 timestamp indicating the start of the interval filter
   --tags value                              tags filter. accepts tagged for all tagged data, untagged for all untagged data, or a list of tags for all data matching any of the tags  (accepts multiple inputs)
```